packages/core/src/tracing/index.js

```diff
@@ -197,6 +198,10 @@ exports.init = function init(_config, downstreamConnection, _processIdentityProv
   automaticTracingEnabled = config.tracing.automaticTracingEnabled;
 
   if (tracingEnabled) {
+    const loader = tracerInstrumentationInfo();
```

Comment:
IMO: This should be called only in debug mode. Could you explore the options to do that?

Reply:
If there is an issue reported and we want to check the logs, then IMO it might be helpful to know the instrumentation details. 
But for that, we can do `console.info` instead of `console.debug`!
Whats your take on that?

Reply:
IMO this should only be logged if debug mode is enabled.